awx_collection/README.md

```diff
@@ -101,6 +99,24 @@ The following notes are changes that may require changes to playbooks:
  - `tower_credential` no longer supports passing a file name to `ssh_key_data`.
  - The HipChat `notification_type` has been removed and can no longer be created using the `tower_notification_template` module.
 
+## Frequently Asked Questions
+### What are Ansible Collections?
+- Docs: Ansible Dev Guide > Developing Collections: https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html
+
+### What are AWX and Ansible Tower?
+- Source: https://github.com/ansible/awx :
+  > AWX provides a web-based user interface, REST API, and task engine built on top of Ansible. It is one of the upstream projects for Red Hat Ansible Automation Platform.
+  - AWX is an open source web app written in Python with Django REST Framework (DRF) and ReactJS for running Ansible tasks (as playbooks, roles, and collections thereof) on inventoried nodes with ansible-runner and searchably logging output.
+- Docs: https://docs.ansible.com/ansible-tower/
+- Docs: https://docs.ansible.com/ansible-tower/latest/html/installandreference/
+- Docs: https://docs.ansible.com/ansible-tower/latest/html/towerapi/
+- Docs: https://docs.ansible.com/ansible-tower/latest/html/towercli/
+- Docs: https://docs.ansible.com/ansible-tower/latest/html/userguide/
+- Docs: https://docs.ansible.com/ansible-tower/latest/html/administration/
+- Docs: https://docs.ansible.com/ansible-tower/latest/html/administration/logging.html
+- Docs: https://docs.ansible.com/ansible-tower/latest/html/administration/security_best_practices.html
+- Docs: https://www.ansible.com/products/awx-project/faq
```

Comment:
I can understand the need for some _minimal_ description and reference for people to know what AWX is, but these doc links are missing relevance to the collection. See here:

https://docs.ansible.com/ansible/latest/collections/awx/awx/index.html

I would be in favor of adding that link, which is a public rendering of the plugin-level documentation of this collection.

Reply:
Yeah I think https://docs.ansible.com/ansible/latest/collections/awx/awx/index.html should be up top in the README(s), too